src/QsCompiler/CompilationManager/CompilationUnit.cs

```diff
@@ -44,7 +44,7 @@ public class Headers
                     ?? ImmutableArray<(SpecializationDeclarationHeader, SpecializationImplementation)>.Empty;
             }
 
-            internal Headers(NonNullable<string> source, IEnumerable<QsNamespace> syntaxTree) : this (
+            public Headers(NonNullable<string> source, IEnumerable<QsNamespace> syntaxTree) : this (
```

Comment:
If reasonable, I'd suggest an API documentation comment here would be helpful now that this constructor is public.

Reply:
Yes, agreed.